consensus/istanbul/backend/api.go

```diff
@@ -214,41 +214,61 @@ func (api *APIExtension) GetCommitteeSize(number *rpc.BlockNumber) (int, error)
 	}
 }
 
-func (api *APIExtension) getProposerAndValidators(block *types.Block) (common.Address, []common.Address, error) {
+type ConsensusInfo struct {
+	proposer       common.Address
+	originProposal common.Address
+	committee      []common.Address
+	round          byte
+}
+
+func (api *APIExtension) getConsensusInfo(block *types.Block) (ConsensusInfo, error) {
 	blockNumber := block.NumberU64()
 	if blockNumber == 0 {
-		return common.Address{}, []common.Address{}, nil
+		return ConsensusInfo{}, nil
 	}
 
+	round := block.Header().Round()
 	view := &istanbul.View{
 		Sequence: new(big.Int).Set(block.Number()),
-		Round:    new(big.Int).SetInt64(0),
+		Round:    new(big.Int).SetInt64(int64(round)),
```

Comment:
This change needs to be informed to node operators since it can change old API outputs

Reply:
Right. We should announce it when we release this.